frontend/src/fixtures/courseFixtures.js

```diff
@@ -0,0 +1,37 @@
+const coursesFixtures = {
+  twoCourses:
+    [
+      {
+      "quarter": "20211",
+      "courseId": "CMPSC     8  ",
```

Comment:
I don't think it matters but, "CMPSC" and "8" get a divorce or smt? why they so far apart

Reply:
That's how it is when you get call the API I think

Reply:
yes fix the spacing

Reply:
NO!  Don't fix the spacing.  The spacing is actually supposed to be this way.

The reason is that the spacing is significant; it ensures that CMPSC 8 comes before CMPSC 130A when you sort by course number.